containers/kubernetes/how-to/revoke-kubeconfig.mdx

```diff
@@ -0,0 +1,71 @@
+---
+meta:
+  title: How to revoke a Kubeconfig of a Kubernetes cluster
+  description: This page explains how to revoke a Kubeconfig of a Kubernetes cluster
+content:
+  h1: How to revoke a Kubeconfig of a Kubernetes cluster
+  paragraph: This page explains how to revoke a Kubeconfig of a Kubernetes cluster
+tags: kubernetes cluster kubeconfig revoke iam
+dates:
+  validation: 2023-11-28
```

Comment:
maybe update?

Reply:
No, it should be revoke / invalidate - do you know any better word?

Reply:
I meant update the validation date :joy: